src/ConvertToSARIF.Engine/RuleSuppression.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I would suggest using the built-in `Suppression` class from `Microsoft.CodeAnalysis.Sarif`. So this extra class is unneeded